marklogic-data-hub/src/main/java/com/marklogic/hub/step/impl/IngestionStepDefinitionImpl.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2012-2019 MarkLogic Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.marklogic.hub.step.impl;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.marklogic.hub.step.AbstractStepDefinition;
+import com.marklogic.hub.step.StepDefinition;
+import com.marklogic.hub.util.json.JSONObject;
+
+import java.util.Map;
+
+public class IngestionStepDefinitionImpl extends AbstractStepDefinition {
+
+    private JsonNode fileLocations;
+
+    public IngestionStepDefinitionImpl(String name) {
+        setName(name);
+        setType(StepDefinitionType.INGEST);
+        setFileLocations(fileLocations);
+
+        Map<String, Object> options = getOptions();
+        options.put("outputFormat", "json");
+
+        setModulePath("/data-hub/5/builtins/steps/ingest/default/main.sjs");
+    }
```

Comment:
Should this module path be changed and similarly for mastering, mapping etc ?

Reply:
Yes, I've added 
`// TODO: Rename internal artifacts step folder to step-definitions, change collections URI` 
for that change as I dont want to break the enode code.
Should be taken up as part of https://project.marklogic.com/jira/browse/DHFPROD-2096